cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedCall.qll

```diff
@@ -364,10 +364,14 @@ class TranslatedFunctionCall extends TranslatedCallExpr, TranslatedDirectCall {
 
   final override predicate mayThrowException() {
     expr.getTarget().(ThrowingFunction).mayThrowException(_)
+    or
+    expr.getTarget() instanceof AlwaysSehThrowingFunction
   }
```

Comment:
I think we should be able to remove this predicate completely, but let's leave that as a follow up.

Reply:
Leaving it until the next iteration.